dependencies.yaml

```diff
@@ -301,6 +301,7 @@ dependencies:
           - pytest-cov>=2.12.1
           - pytest-lazy-fixtures>=1.0.0
           - pytest-xdist
+          - setuptools>=60.0  # provides 'distutils', keeps 'GPUtil' working on Python 3.12+
```

Comment:
`setuptools==60.0.0` is the first release where `disutils` was vendored. From https://setuptools.pypa.io/en/latest/deprecated/distutils-legacy.html

> *Since the 60.0.0 release, Setuptools includes a local, vendored copy of distutils (from late copies of CPython) that is enabled by default.*

That release is from December 2021 ([link](https://pypi.org/project/setuptools/60.0.0/)), so I'd be surprised if it causes any conflicts.